scripts/byron-to-alonzo/mkfiles.sh

```diff
@@ -583,23 +583,31 @@ echo "$ROOT/run/all.sh"
 echo
 echo "In order to do the protocol updates, proceed as follows:"
 echo
-echo "  0. wait for the nodes to start producing blocks"
-echo "  1. invoke ./scripts/byron-to-alonzo/update-1.sh"
+echo "  0. invoke ./scripts/byron-to-alonzo/mkfiles.sh"
```

Comment:
I think this needs to be clearer because it needs to be invoked with an argument that is the latest era.